### DISCUSSION TITLE
Using inset and custom color with box shadow.

### DISCUSSION THREAD

**Discussion by gajrajantino** [tailwindlabs/tailwindcss#8581] [Help] [answered]
I wanna do something like `shadow-[inset_0px_0px_0px_2px_blue-custom-900]` i.e. an inset box shadow with color `blue-custom-900` that I have defined in my `tailwind.config.js`. Is it possible to do so ? Or please suggest an alternative, I am using React :-)

**Comment by gajrajantino** [comment]
I also tried this, doesn't work. afaik this was introduced in latest tailwind (v3.1)

```jsx
style={{boxShadow: "inset 0px 0px 0px 2px theme(colors.blue-custom.900)" }}
```

**Comment by crswll** [comment, marked as answer]
`style` wouldn't know what to do with `theme`. You can do `class="shadow-[inset_0_0_0_2px_theme(colors.blue-custom.900)]"` though.

**Comment by gajrajantino** [comment]
Thanks! :)
